desktop/cmp/treemap/TreeMap.js

```diff
@@ -0,0 +1,297 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+import {Component} from 'react';
+import PT from 'prop-types';
+import {Highcharts, highchartsExporting, highchartsOfflineExporting, highchartsExportData, highchartsTree, highchartsHeatmap} from '@xh/hoist/kit/highcharts';
+
+import {XH, elemFactory, HoistComponent, LayoutSupport} from '@xh/hoist/core';
+import {div, box} from '@xh/hoist/cmp/layout';
+import {Ref} from '@xh/hoist/utils/react';
+import {resizeSensor} from '@xh/hoist/kit/blueprint';
+import {fmtNumber} from '@xh/hoist/format';
+import {forEachAsync} from '@xh/hoist/utils/async';
+import {Cube} from '@xh/hoist/data/cube';
+import {assign, merge, clone, debounce} from 'lodash';
+
+import {LightTheme} from './theme/Light';
+import {DarkTheme} from './theme/Dark';
+
+import './TreeMap.scss';
+import {TreeMapModel} from './TreeMapModel';
+
+highchartsExporting(Highcharts);
+highchartsOfflineExporting(Highcharts);
+highchartsExportData(Highcharts);
+highchartsTree(Highcharts);
+highchartsHeatmap(Highcharts);
```

Comment:
Should we be doing these in the `/kit/` index file?  Or at least the ones we are also doing in `Chart.js`?  I assume they are initializing / enabling these features/plugins.  Could it be an issue if this is done twice?  

If there is some cost (in terms of running the code or roping additional libraries into the webpack bundle) I can see keeping the two map-related calls here in this component.